### PR TITLE
docs: fix the license URLs of client_model, flatbuffers and hyperscan

### PR DESCRIPTION
**Pull request by mkauf**
Risk Level: Low
Docs Changes: Fix license URLs

### CODE REVIEW

File: bazel/repository_locations.bzl [hunk 1]
```diff
@@ -1196,7 +1196,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2023-03-03",
         cpe = "cpe:2.3:a:google:flatbuffers:*",
         license = "Apache-2.0",
-        license_url = "https://github.com/google/flatbuffers/blob/v{version}/LICENSE.txt",
```
Comment (phlax) [inline bazel/repository_locations.bzl:1199]: this one has been renamed -> `LICENSE` (including in our current version)

Comment (mkauf) [reply bazel/repository_locations.bzl:1199]: That's correct. Please check my commit again - the new name "LICENSE" is used.

Ok? (or maybe I did not understand your comment)

Comment (phlax) [reply bazel/repository_locations.bzl:1199]: my bad - i read it the wrong way round

**Review by phlax — APPROVED** [review]
lgtm, thanks @mkauf

**Comment by mkauf** [comment]
Thank you for reviewing and merging!

**Pull request merged**